### ISSUE TITLE
reinitialize state and action encoder

### ISSUE THREAD

**Issue by yfpeng1234** [Physical-Intelligence/openpi#467]
Dear author,
Thanks for opensource this work. I try to deploy pi0 on my own robot. I re-initialize the state and action encoder for flexibility, as they are simply mlp. But after lora finetuning, I found the final loss is around 0.26. And I also sample observation from my dataset and compare the predicted action with ground truth. The error is very big. Should I follow the setting of finetuning on libero, where the state and action encoder are initialized from pretrained ckpt?
Best

**Comment by kpertsch** [comment]
It generally seems a good idea to initialize them from pre-training, this is how we do all our finetuning!
